### ISSUE TITLE
ACCESSIBILITY: No Screenreader Output in the "Autotext" create/edit dialog

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#11682]
### Steps to reproduce:
1. Open Writer
2. Write a few words
3. Open the Autotext Dialogue
4. Select "My Autotext" and hit the "Autotext" Select Button
7. In the now open Select Menu, the screenreader NVDA is silent.

### Actual behavior:
1. Open Writer
2. Write a few words.
3. Mark a few words
4. Move to the "Extra" -> Autotext Menu item and hit the Menuitem.
5. In the now open dialogue move with the "Tabulator" key to the list of categroriesand select "My Autotext".
6. move on with the tabulator Key to the "Autotext" Select Button and hit it.
7. In the now open Field you an select what to do with your Autotext, but the screenreader NVDA is silent.

### Expected behavior:
The screenreader should announce the items which can be selected.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2020.2

#### Windows version:
Edition	Windows 	10 Home
Version				1909
Operating system build	18363.1016

#### Name and version of other software in use when reproducing the issue:
LibreOffice
Version:	7.0.0.3 (x64)
Build ID:	8061b3e9204bef6b321a21033174034a5e2ea88e
CPU-Threads: 	4; BS: Windows 10.0 Build 18363; UI-Render: Skia/Vulkan; VCL: win
Locale: 	de-DE (de_DE); UI: de-DE
Calc: 	threaded

#### Other information about your system:
-

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
Not tested.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

**Comment by peterpacult** [comment]
The issue has also been reported to the Document Foundation; see https://bugs.documentfoundation.org/show_bug.cgi?id=135423
